### PR TITLE
Updated installation guide for Docker setup

### PR DESCRIPTION
**Pull request by jayam04**

### CODE REVIEW

File: Installing-Oppia.md [hunk 1]
```diff
@@ -17,19 +17,23 @@ following:
 
 If you need to learn or brush up on these, please do so before trying to start the installation so you don't get confused by our instructions.
 
-## Standard installation using Python setup
+## Python setup (soon to be depreciated)
```
Comment (seanlip) [inline Installing-Oppia.md]: Thanks @jayam04 -- can we put this section below the Docker one please?

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @jayam04 -- looks good, just one suggested change, thanks!

File: Installing-Oppia.md (hunk 1 above)
Comment (jayam04) [reply Installing-Oppia.md]: done!

**Comment by jayam04** [comment]
@seanlip moved python setup below docker, PTAL!

**Review by seanlip — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**